ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -241,37 +243,73 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
 
 template <typename ShapeFunction, typename IntegrationMethod,
           unsigned GlobalDim>
-void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
-    computeSecondaryVariableConcrete(double const t,
-                                     std::vector<double> const& local_x)
+std::vector<double> const&
+LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
+    getIntPtDarcyVelocity(const double t,
+                          GlobalVector const& current_solution,
+                          NumLib::LocalToGlobalIndexMap const& dof_table,
+                          std::vector<double>& veloctiy_cache) const
 {
+    // auto const num_nodes = ShapeFunction_::NPOINTS;
+    auto const num_intpts = _shape_matrices.size();
```

Comment:
Minor: If there is `_integration_method`, then this could be `_integration_method.getNumberOfPoints();`.

Reply:
Changed.